nbgdrive/handlers.py

```diff
@@ -149,6 +149,60 @@ def _get_gdrive_auth_url():
     return cleaned_response
 
 
+def pull_from_gdrive(pull_id):
+    """Pulls the directory associated with the path from user's gdrive"""
+    try:
+        gdrive_folders = pull_id.split('/')
+        dir_ids = gdrive_puller(gdrive_folders, "root")
+        if dir_ids[0] == 'error':
+            return {
+                'status': 'error',
+                'message': 'directory does not exist'
+            }
+        for d in dir_ids:
+            command = 'gdrive download -r ' + d
+            p = Popen(command, stdout=PIPE, shell=True)
+            output, err = p.communicate()
+        return {
+            'status': 'success'
+        }
+    except Exception as e:
+        exc_type, exc_obj, exc_tb = sys.exc_info()
+        return {
+            'status': 'error',
+            'message': str(e) + str(exc_tb.tb_lineno)
+        }
+
+
+def gdrive_puller(pull_folders, parent):
```

Comment:
A quick comment (just a stylistic comment), add an `_` in front of this function if it will only be used internally -- i.e it is more of a helper function that one that is exposed via an actual endpoint.